Fibers dependency breaks in node 16

### Describe the problem:

With node v.16.0.0 and while executing `npm run dev` or `npm run build`, the following error occurs:

```
[worker] ## There is an issue with `node-fibers` ##
[worker] `/home/leon/Code/shared-music-db/node_modules/fibers/bin/linux-x64-93-glibc/fibers.node` is missing.
[worker] 
[worker] Try running this to fix the issue: /home/leon/.nvm/versions/node/v16.0.0/bin/node /home/leon/Code/shared-music-db/node_modules/fibers/build
[worker] Error: Cannot find module '/home/leon/Code/shared-music-db/node_modules/fibers/bin/linux-x64-93-glibc/fibers'
[worker] Require stack:
[worker] - /home/leon/Code/shared-music-db/node_modules/fibers/fibers.js
[worker] - /home/leon/Code/shared-music-db/node_modules/flareact/configs/webpack.config.js
[worker] - /home/leon/Code/shared-music-db/node_modules/flareact/configs/webpack.worker.config.js
[worker] - /home/leon/Code/shared-music-db/node_modules/flareact/webpack.js
[worker] - /home/leon/.cache/.wrangler/wranglerjs-1.15.0/index.js
[worker]     at Function.Module._resolveFilename (node:internal/modules/cjs/loader:940:15)
[worker]     at Function.Module._load (node:internal/modules/cjs/loader:773:27)
[worker]     at Module.require (node:internal/modules/cjs/loader:1012:19)
[worker]     at require (node:internal/modules/cjs/helpers:93:18)
[worker]     at Object.<anonymous> (/home/leon/Code/shared-music-db/node_modules/fibers/fibers.js:14:39)
[worker]     at Module._compile (node:internal/modules/cjs/loader:1108:14)
[worker]     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1137:10)
[worker]     at Module.load (node:internal/modules/cjs/loader:988:32)
[worker]     at Function.Module._load (node:internal/modules/cjs/loader:828:14)
[worker]     at Module.require (node:internal/modules/cjs/loader:1012:19)
[worker]     at require (node:internal/modules/cjs/helpers:93:18)
[worker]     at module.exports (/home/leon/Code/shared-music-db/node_modules/flareact/configs/webpack.config.js:29:16)
[worker]     at module.exports (/home/leon/Code/shared-music-db/node_modules/flareact/configs/webpack.worker.config.js:26:8)
[worker]     at /home/leon/.cache/.wrangler/wranglerjs-1.15.0/index.js:43:50
[worker]     at Object.<anonymous> (/home/leon/.cache/.wrangler/wranglerjs-1.15.0/index.js:139:3)
[worker]     at Module._compile (node:internal/modules/cjs/loader:1108:14)
[worker] /home/leon/Code/shared-music-db/node_modules/fibers/fibers.js:23
[worker]                throw new Error('Missing binary. See message above.');
[worker]                      ^
[worker] 
[worker] Error: Missing binary. See message above.
[worker]     at Object.<anonymous> (/home/leon/Code/shared-music-db/node_modules/fibers/fibers.js:23:9)
[worker]     at Module._compile (node:internal/modules/cjs/loader:1108:14)
[worker]     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1137:10)
[worker]     at Module.load (node:internal/modules/cjs/loader:988:32)
[worker]     at Function.Module._load (node:internal/modules/cjs/loader:828:14)
[worker]     at Module.require (node:internal/modules/cjs/loader:1012:19)
[worker]     at require (node:internal/modules/cjs/helpers:93:18)
[worker]     at module.exports (/home/leon/Code/shared-music-db/node_modules/flareact/configs/webpack.config.js:29:16)
[worker]     at module.exports (/home/leon/Code/shared-music-db/node_modules/flareact/configs/webpack.worker.config.js:26:8)
[worker]     at /home/leon/.cache/.wrangler/wranglerjs-1.15.0/index.js:43:50
[worker]     at Object.<anonymous> (/home/leon/.cache/.wrangler/wranglerjs-1.15.0/index.js:139:3)
[worker]     at Module._compile (node:internal/modules/cjs/loader:1108:14)
[worker]     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1137:10)
[worker]     at Module.load (node:internal/modules/cjs/loader:988:32)
[worker]     at Function.Module._load (node:internal/modules/cjs/loader:828:14)
[worker]     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:76:12)
[worker] Error: failed to execute `"/home/leon/.nvm/versions/node/v16.0.0/bin/node" "/home/leon/.cache/.wrangler/wranglerjs-1.15.0" "--output-file=/tmp/.wranglerjs_outputx5SQc" "--wasm-binding=WASM_MODULE" "--webpack-config=node_modules/flareact/webpack"`: exited with exit code: 1
[worker] wrangler dev exited with code 1
--> Sending SIGTERM to other processes..
[client] ## There is an issue with `node-fibers` ##
[client] `/home/leon/Code/shared-music-db/node_modules/fibers/bin/linux-x64-93-glibc/fibers.node` is missing.
[client] 
[client] Try running this to fix the issue: /home/leon/.nvm/versions/node/v16.0.0/bin/node /home/leon/Code/shared-music-db/node_modules/fibers/build
[client] Error: Cannot find module '/home/leon/Code/shared-music-db/node_modules/fibers/bin/linux-x64-93-glibc/fibers'
[client] Require stack:
[client] - /home/leon/Code/shared-music-db/node_modules/fibers/fibers.js
[client] - /home/leon/Code/shared-music-db/node_modules/flareact/configs/webpack.config.js
[client] - /home/leon/Code/shared-music-db/node_modules/flareact/configs/webpack.client.config.js
[client] - /home/leon/Code/shared-music-db/node_modules/webpack-cli/bin/utils/convert-argv.js
[client] - /home/leon/Code/shared-music-db/node_modules/webpack-dev-server/bin/webpack-dev-server.js
[client]     at Function.Module._resolveFilename (node:internal/modules/cjs/loader:940:15)
[client]     at Function.Module._load (node:internal/modules/cjs/loader:773:27)
[client]     at Module.require (node:internal/modules/cjs/loader:1012:19)
[client]     at require (node:internal/modules/cjs/helpers:93:18)
[client]     at Object.<anonymous> (/home/leon/Code/shared-music-db/node_modules/fibers/fibers.js:14:39)
[client]     at Module._compile (node:internal/modules/cjs/loader:1108:14)
[client]     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1137:10)
[client]     at Module.load (node:internal/modules/cjs/loader:988:32)
[client]     at Function.Module._load (node:internal/modules/cjs/loader:828:14)
[client]     at Module.require (node:internal/modules/cjs/loader:1012:19)
[client] /home/leon/Code/shared-music-db/node_modules/fibers/fibers.js:23
[client]                throw new Error('Missing binary. See message above.');
[client]                ^
[client] 
[client] Error: Missing binary. See message above.
[client]     at Object.<anonymous> (/home/leon/Code/shared-music-db/node_modules/fibers/fibers.js:23:9)
[client]     at Module._compile (node:internal/modules/cjs/loader:1108:14)
[client]     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1137:10)
[client]     at Module.load (node:internal/modules/cjs/loader:988:32)
[client]     at Function.Module._load (node:internal/modules/cjs/loader:828:14)
[client]     at Module.require (node:internal/modules/cjs/loader:1012:19)
[client]     at require (node:internal/modules/cjs/helpers:93:18)
[client]     at module.exports (/home/leon/Code/shared-music-db/node_modules/flareact/configs/webpack.config.js:29:16)
[client]     at module.exports (/home/leon/Code/shared-music-db/node_modules/flareact/configs/webpack.client.config.js:86:8)
[client]     at handleFunction (/home/leon/Code/shared-music-db/node_modules/webpack-cli/bin/utils/prepareOptions.js:21:13)
[client] webpack-dev-server --config node_modules/flareact/configs/webpack.client.config.js --mode development exited with code 1
[
  {
    command: {
      command: 'wrangler dev',
      name: 'worker',
      prefixColor: '',
      env: [Object]
    },
    index: 0,
    exitCode: 1
  },
  {
    command: {
      command: 'webpack-dev-server --config node_modules/flareact/configs/webpack.client.config.js --mode development',
      name: 'client',
      prefixColor: '',
      env: [Object]
    },
    index: 1,
    exitCode: 1
  }
]
```

Executing `/home/leon/.nvm/versions/node/v16.0.0/bin/node /home/leon/Code/shared-music-db/node_modules/fibers/build` as opposed results in an `node-gyp` error code 1 and the hint to use a supported platform and node version.

When switching to node v.15.14.0 (via `nvm use 15`), everything works fine. This happens with two of my own flareact projects.

### Link to a minimal reproduction:

Just take any Flareact Repo and execute `npm i` and `npm run dev` when using node v16.0.0.

**EDIT:** It seems like `node-fibers` actually stopped working as of April 13th, 2021 as mentioned in the README.md of https://github.com/laverdet/node-fibers. It does not seems like that the project will be maintained any longer.

--- a/configs/webpack.config.js
+++ b/configs/webpack.config.js
@@ -21,7 +21,7 @@ module.exports = function ({ dev, isServer }) {
     loader: MiniCssExtractPlugin.loader,
   };
 
-  const sassOptionFiber = false;
+  let sassOptionFiber = false;
 
   // Check if we can enable fibers or not. Its not supported in node versions 16 and up.
   if (parseInt(process.versions.node.split('.')[0]) < 16) {

--- a/configs/webpack.config.js
+++ b/configs/webpack.config.js
@@ -21,12 +21,19 @@ module.exports = function ({ dev, isServer }) {
     loader: MiniCssExtractPlugin.loader,
   };
 
+  const sassOptionFiber = false;
+
+  // Check if we can enable fibers or not. Its not supported in node versions 16 and up.
+  if (parseInt(process.versions.node.split('.')[0]) < 16) {
+    sassOptionFiber = require("fibers");
+  }
+
   const sassLoader = {
     loader: "sass-loader",
     options: {
       implementation: require("sass"),
       sassOptions: {
-        fiber: require("fibers"),
+        fiber: sassOptionFiber
       },
     },
   };

